app/views/products/show.html.erb

```diff
@@ -0,0 +1,12 @@
+<div class = "product-show">
+    <div class = "product-details">
+        <%= @product.name %>
+    </div>
```

Comment:
Don't forget to separate your closing html elements from opening ones with a blank line!